app/templates/_react.html

```diff
@@ -0,0 +1,11 @@
+{% extends "_base_page.html" %}
+
+{% block body_end %}
+  <script type="text/javascript" src="{{ component.get_vendor_bundle() }}"></script>
+  <script type="text/javascript" src="{{ component.get_bundle() }}"></script>
+{% endblock %}
+
+{% block main_content %}
```

Comment:
This will need to be updated to match https://github.com/AusDTO/dto-digitalmarketplace-buyer-frontend/pull/383